src/container/RelayRootContainer.js

```diff
@@ -27,6 +28,7 @@ type RootContainerProps = {
   Component: RelayContainer;
   forceFetch?: ?boolean;
   onReadyStateChange?: ?(readyState: ReadyState) => void;
+  relayContext: RelayContext;
```

Comment:
We definitely want `RelayRenderer` to accept a `RelayContext` as a prop. However, we also need to make the transition easy for developers. Let's do this:
- Keep the addition of `relayContext` to RelayRenderer.
- Add `RelayContext.getDefaultInstance()` (static) and have `RelayRootContainer` pass that default instance as the context to RelayRenderer.